### ISSUE TITLE
Client App - Npm start starts the server but shows error in browser console - screenshot and details added

### ISSUE THREAD

**Issue by gunjank** [souravbh/mockProxyServer#3]
checkPropTypes.js?5679:19 Warning: Failed prop type: Invalid prop `caret` of type `string` supplied to `DropdownToggle`, expected `boolean`.
    in DropdownToggle (created by TopNavBar)
    in TopNavBar (created by LandingLayout)
    in div (created by LandingLayout)
    in LandingLayout (created by LoadableComponent)
    in LoadableComponent (created by Route)
    in Route (created by App)
    in Switch (created by App)
    in div (created by App)
    in App
    in Router (created by ConnectedRouter)
    in ConnectedRouter (created by Connect(ConnectedRouter))
    in Connect(ConnectedRouter)
    in IntlProvider (created by LanguageProvider)
    in LanguageProvider (created by Connect(LanguageProvider))
    in Connect(LanguageProvider)
    in Provider
printWarning @ checkPropTypes.js?5679:19
checkPropTypes @ checkPropTypes.js?5679:82
validatePropTypes @ react.development.js?72d0:1568
createElementWithValidation @ react.development.js?72d0:1657
render @ index.js?0ff0:41
finishClassComponent @ react-dom.development.js?61bb:13727
updateClassComponent @ react-dom.development.js?61bb:13690
beginWork @ react-dom.development.js?61bb:14489
performUnitOfWork @ react-dom.development.js?61bb:17014
workLoop @ react-dom.development.js?61bb:17054
renderRoot @ react-dom.development.js?61bb:17133
performWorkOnRoot @ react-dom.development.js?61bb:17993
performWork @ react-dom.development.js?61bb:17901
performSyncWork @ react-dom.development.js?61bb:17873
requestWork @ react-dom.development.js?61bb:17761
scheduleWork @ react-dom.development.js?61bb:17566
enqueueSetState @ react-dom.development.js?61bb:11695
Component.setState @ react.development.js?72d0:373
safeSetState @ loadable-components.es.js?c034:216
(anonymous) @ loadable-components.es.js?c034:198
Promise.then (async)
LoadableComponent @ loadable-components.es.js?c034:197
constructClassInstance @ react-dom.development.js?61bb:11864
updateClassComponent @ react-dom.development.js?61bb:13681
beginWork @ react-dom.development.js?61bb:14489
performUnitOfWork @ react-dom.development.js?61bb:17014
workLoop @ react-dom.development.js?61bb:17054
renderRoot @ react-dom.development.js?61bb:17133
performWorkOnRoot @ react-dom.development.js?61bb:17993
performWork @ react-dom.development.js?61bb:17901
performSyncWork @ react-dom.development.js?61bb:17873
requestWork @ react-dom.development.js?61bb:17761
scheduleWork @ react-dom.development.js?61bb:17566
scheduleRootUpdate @ react-dom.development.js?61bb:18240
updateContainerAtExpirationTime @ react-dom.development.js?61bb:18267
updateContainer @ react-dom.development.js?61bb:18324
ReactRoot.render @ react-dom.development.js?61bb:18586
(anonymous) @ react-dom.development.js?61bb:18726
unbatchedUpdates @ react-dom.development.js?61bb:18124
legacyRenderSubtreeIntoContainer @ react-dom.development.js?61bb:18722
render @ react-dom.development.js?61bb:18783
render @ app.js?9fee:42
(anonymous) @ app.js?9fee:75
./app/app.js @ main.js:939
__webpack_require__ @ main.js:788
fn @ main.js:151
0 @ main.js:1128
__webpack_require__ @ main.js:788
checkDeferredModules @ main.js:46
(anonymous) @ main.js:926
(anonymous) @ main.js:929
checkPropTypes.js?5679:19 Warning: Failed prop type: Invalid prop `nav` of type `string` supplied to `DropdownToggle`, expected `boolean`.
    in DropdownToggle (created by TopNavBar)
    in TopNavBar (created by LandingLayout)
    in div (created by LandingLayout)
    in LandingLayout (created by LoadableComponent)
    in LoadableComponent (created by Route)
    in Route (created by App)
    in Switch (created by App)
    in div (created by App)
    in App
    in Router (created by ConnectedRouter)
    in ConnectedRouter (created by Connect(ConnectedRouter))
    in Connect(ConnectedRouter)
    in IntlProvider (created by LanguageProvider)
    in LanguageProvider (created by Connect(LanguageProvider))
    in Connect(LanguageProvider)
    in Provider

![image](https://user-images.githubusercontent.com/1900029/53539074-1b7f7780-3ade-11e9-90a2-b7753873f345.png)

**Comment by souravbh** [comment]
The issue happens for collision of react bootstrap with the current binding. The error in console will not cause any behavioral issues in client. The react bootstrap will soon be de commissioned from the app. Are you facing any navigational or loading issue in client?

**Comment by gunjank** [comment]
Yes most of the UI controls are disabled also not able to add swagger to mock.


Sent from Yahoo Mail for iPhone


On Thursday, February 28, 2019, 8:00 AM, Sourav <notifications@github.com> wrote:


The issue happens for collision of react bootstrap with the current binding. The error in console will not cause any behavioral issues in client. The react bootstrap will soon be de commissioned from the app. Are you facing any navigational or loading issue in client?

**Comment by souravbh** [comment]
The issue happens when a wrong swagger file is uploaded. Will be fixed soon.
